starbucks/src/main/java/git_kkalnane/backend/starbucks/item/domain/BervergeItem.java

```diff
@@ -0,0 +1,49 @@
+package git_kkalnane.backend.starbucks.item.domain;
+
+import git_kkalnane.backend.starbucks.global.entity.BaseTimeEntity;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@Entity
+@Table(name = "beverage_items")
+@NoArgsConstructor
+public class BervergeItem extends BaseTimeEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "Beverage_item_name_ko", nullable = false, unique = true, length = 50)
+    private String itemNameKo;
+
+    @Column(name = "item_name_en", nullable = false, unique = true, length = 50)
```

Comment:
위의 컬럼명과 함께 일관적으로 작명 해주시면 착오가 없을 듯 합니다.

```suggestion
    @Column(name = "beverage_item_name_en", nullable = false, unique = true, length = 50)
```

Reply:
미처 하나만 바꾸고 나머지 바꾸지 못했네요.. dessert도 함께 수정 완료헀습니다!